CONTRIBUTING.md

```diff
@@ -28,47 +31,57 @@ See the [Eclipse Code Of Conduct](https://www.eclipse.org/org/documents/Communit
 Before your contribution can be accepted by the project, you need to create and electronically sign
 a [Eclipse Contributor Agreement (ECA)](http://www.eclipse.org/legal/ecafaq.php):
 
-1. Log in to the [Eclipse foundation website](https://accounts.eclipse.org/user/login/). You will need to create an
-   account with the Eclipse Foundation if you have not already done so.
+1. Log in to the [Eclipse foundation website](https://accounts.eclipse.org/user/login/). You will 
+   need to create an account within the Eclipse Foundation if you have not already done so.
 2. Click on "Eclipse ECA", and complete the form.
 
-Be sure to use the same email address in your Eclipse Account that you intend to use when you commit to GitHub.
+Be sure to use the same email address in your Eclipse Account that you intend to use when you commit 
+to GitHub.
 
 ## How to Contribute
 
 ### Discuss
 
-If you feel there is a bug or an issue contribute to discussions in
-[existing issues](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/issues?q=is%3Aissue+is%3Aopen)
-otherwise [create a new issue](#create-an-issue).
+If you feel there is a bug or an issue, contribute to the discussions in
+[existing issues](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/issues?q=is%3Aissue+is%3Aopen),
+otherwise [create a new issue](#create-an-issue). If you want to share an idea to further enhance 
```

Comment:
I would put the "discussions" part first, since it's better to incentivize discussions more than issues